Xolof/fishlog-react
# Sign up

### Result
The user can now create an account.

### How it was done
A component Signup.js was added in which requests can be made to the APIs route /api/register.

### How to verify
Click "Login" in the menu.
Click the link "Create new account" below the form.
Fill out and submit the form.
Try with an invalid email-address, a very short password or a username that's already been taken. Then you should get error messages.

### Before merge
You can run the end to end test with `npm run cypress`. Choose end-to-en-tests.js.
